Sources/RedisMulti+Set.swift

```diff
@@ -0,0 +1,590 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/// Extend Redis by adding the Set operations
+extension RedisMulti {
+    
+    /// Add one or more members to a set
+    ///
+    /// - Parameter key: The key.
+    /// - Parameter memebers: The values to be added to the set.
```

Comment:
Please add the - Returns: clause to the API docs, this applies to all of the APIs here.